exchanges/order/orders.go

```diff
@@ -1234,6 +1254,37 @@ func StringToOrderStatus(status string) (Status, error) {
 	}
 }
 
+// StringToTimeInForce converts time in force string value to TimeInForce instance.
+func StringToTimeInForce(timeInForce string) (TimeInForce, error) {
```

Comment:
suggestion: This is a good easy catch all but it adds bloat and reduced performance there should be an umarshal json method or function to a type exchange side. So suggestion would be to remove this completely. This is just a suggestion doesn't need to be done.

Reply:
Let's keep it here and remove it for when we need the UnmarshalJSON method rather than this one